tests/Integration/Behaviour/Features/Scenario/ApiAccess/api_access_management.feature

```diff
@@ -0,0 +1,31 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s api-access --tags api-access-management
+@restore-api-access-before-feature
+@api-access-management
+Feature: Api Access Management
+  PrestaShop provides an API to manage and export BO data
+  As a API user
+  I must be able to create, save and edit api access
+
+  Scenario: Create a simple api access and edit it
+    When I create an api-access "AA-1" with following properties:
+      | clientName   | Thomas               |
+      | apiClientId  | test-id              |
+      | enabled      | true                 |
+      | description  | a simple description |
+    When I query api-access "AA-1" I should get a ApiAccess with properties:
+      | clientName   | Thomas               |
+      | apiClientId  | test-id              |
+      | enabled      | true                 |
+      | description  | a simple description |
+    When I edit api-access "AA-1" and I change the following properties:
+      | clientName | Tom |
+    When I query api-access "AA-1" I should get a ApiAccess with properties:
+      | clientName | Tom |
+
+  Scenario: Create an api-access with invalid properties:
+    When I create an api-access "AA-2" with following properties:
+      | clientName   | Thomas               |
+      | apiClientId  | test-id              |
+      | enabled      | true                 |
+      | description  | a simple description |
+    Then I should get an error that fields are not unique
```

Comment:
That's a good first test, you will have to add the same principle for each property that has validation constraints to force their invalid state and check that the proper exception is thrown